accounts-db/src/accounts_db.rs

```diff
@@ -5082,6 +5104,41 @@ impl AccountsDb {
             }
         };
 
+        let mut limit = 10;
```

Comment:
@HaoranYi @brooksprumo this is the algorithm that I hacked up to have shrink pick up some shrinkable ancient storages if shrink isn't overloaded. I don't love this, but it appears to work.